### DISCUSSION TITLE
URL redirect support method for Authenticated URL

### DISCUSSION THREAD

**Discussion by Zinfandel22** [schreibfaul1/ESP32-audioI2S#474] [Q&A]
One station I am trying to stream has a fixed URL that has an authentication key.
A connection to the stream provider provides a new Key.

Seem this library is not accepting the new redirect and new URL

Is a method available to pick up and then stream from the new URL provided on authentication?
void audio_lasthost(const char *info)?

<Radio URL> -->> <Streamguys+dynamickey>

**Comment by schreibfaul1** [comment]
This is not possible. There is only one direct link between URL, Name and PW. You can copy the forwarded URL and then call it with the fitting credentials.
